StripeUICore/StripeUICore/Source/Controls/OneTimeCodeTextField.swift

```diff
@@ -88,9 +133,12 @@ import UIKit
     public lazy var tokenizer: UITextInputTokenizer = UITextInputStringTokenizer(textInput: self)
 
     // MARK: -
-    public init(numberOfDigits: Int = 6, theme: ElementsUITheme = .default) {
-        self.numberOfDigits = numberOfDigits
-        self.textStorage = TextStorage(capacity: numberOfDigits)
+    public init(
+        configuration: Configuration = Configuration(),
+        theme: ElementsUITheme = .default
```

Comment:
it likely makes sense to remove `theme` and put it into `configuration` but taking it one-step-at-a-time and this current PR should get the job done for now